Modules/_cursesmodule.c

```diff
@@ -932,8 +932,9 @@ PyCursesWindow_dealloc(PyObject *self)
     PyObject_GC_UnTrack(self);
     PyCursesWindowObject *wo = (PyCursesWindowObject *)self;
     if (wo->win != stdscr && wo->win != NULL) {
-        // silently ignore errors in delwin(3)
-        (void)delwin(wo->win);
+        if (delwin(wo->win) == ERR) {
+            PyErr_FormatUnraisable("Exception ignored in delwin()");
```

Comment:
Does delwin() function raise an exception? PyErr_FormatUnraisable() requires that an exception is set.

Reply:
So I'll chgange this

Reply:
FormatUnraisable() purpose is to log an exception, so you should set an exception to call it. Yes, you can call PyErr_SetString() for example.